client-ui/src/app/app.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function app() {
```

Comment:
Always keep the name of Custom component pascal case.
```suggestion
function App() {
```